templates/components/search/display_data.html.twig

```diff
@@ -36,27 +36,31 @@
 {% if not is_ajax %}
    <div class="ajax-container search-display-data">
 {% endif %}
+{% set push_history = push_history is defined and push_history %}
+{% set readonly = readonly is defined and readonly %}
```

Comment:
Maybe `hide_controls` would be more appropriate than `readonly`.

Reply:
Done 46ba760ad6.